lib/data_update_scripts/20210824202334_backfill_forem_owner_role.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class BackfillForemOwnerRole
+    def run
+      User.with_role(:super_admin).first.add_role(:forem_owner)
```

Comment:
In the odd chance that this is run on an empty Forem (could be if a person updates the code before they setup their own Forem), this should take that into account:

```ruby
        User.with_role(:super_admin).first&.add_role(:forem_owner)
```